### ISSUE TITLE
file dependency format error

### ISSUE THREAD

**Issue by pabl0rg** [cbeust/kobalt#304]
I have been using this line in my Build.kt for months without issue.

```
    dependencies {
        compile(file("lib/RXTXcomm.jar"))
        compile(file("lib/commons-logging-1.1.jar"))
    }
````

With the more recent builds of Kobalt I get this error:

```
*****
***** ERROR Error: java.lang.IllegalArgumentException: Bad artifact coordinates file://./lib/commons-logging-1.1.jar, expected format is <groupId>:<artifactId>[:<extension>[:<classifier>]]:<version>
*****
```

**Comment by pabl0rg** [comment]
i suspect the lib/ has something to do with this

**Comment by cbeust** [comment]
Fixed in 0.900.
